src/main/java/edu/rosehulman/quota/Quota.java

```diff
@@ -3,33 +3,34 @@
 import java.util.List;
 
 public class Quota {
-	private String id;
-	private String name;
-	private String type; // TODO: Maybe change this to an enum or use the strategy pattern
-	private List<Tier> tiers;
-		
-	public Quota(String quotaId) {
-		this.id = quotaId;
-	}
-
-	public Quota() {
-	}
-
-	public String getID() {
-		return this.id;
-	}
-	
-	
-	@Override
-	public String toString() {
-		String toReturn = "Quota: "  + id + "\n";
-		// tiers?
-		/*for(String id : this.quotaMap.keySet()) {
-			toReturn += this.quotaMap.get(id).toString();
-		}*/
-		return toReturn;
-	}
+  private String id;
+  private String name;
+  private String type; // TODO: Maybe change this to an enum or use the strategy
+                       // pattern
```

Comment:
Same comment as below, we should not have a line limit; It will make our code too long having so many line breaks because of comments wrapping.